libs/dh/admin/feature-invite-user-modal/src/lib/dh-invite-user-modal.component.ts

```diff
@@ -119,13 +109,40 @@ export class DhInviteUserModalComponent implements AfterViewInit {
   emailExists = false;
   knownEmails: string[] = [];
   isLoadingEmails = true;
-  isLoadingAssociatedActors = false;
-  isAlreadyAssociatedToActor = false;
   currentEmail?: string;
 
   baseInfo = this.formBuilder.group({
     actorId: ['', Validators.required],
-    email: [{ value: '', disabled: true }, [Validators.required, Validators.email]],
+    email: [
+      { value: '', disabled: true },
+      [Validators.required, Validators.email],
+      [
+        () => {
+          return this.apollo
+            .mutate({
+              mutation: GetAssociatedActorsDocument,
+              variables: {
+                email: this.currentEmail ?? '',
```

Comment:
If check before, if email is empty, instead of making a call to BFF.

Reply:
Fixed